GitVersionCore.Tests/IntegrationTests/GitHubFlow/ReleaseBranchTests.cs

```diff
@@ -71,7 +75,7 @@ public void WhenReleaseBranchIsMergedIntoMasterHighestVersionIsTakenWithIt()
             fixture.Repository.Checkout("master");
             fixture.Repository.MergeNoFF("release-1.0.0", Constants.SignatureNow());
 
-            fixture.AssertFullSemver("2.0.0+11");
+            fixture.AssertFullSemver("2.0.0+5");
```

Comment:
I assume this is because we are not as smart with commit counting. It is counting commits along the branch, not including the commits which were merged in. Is this an issue?